lib/mapreduce/index.js

```diff
@@ -680,6 +682,10 @@ function queryViewInQueue(view, opts) {
     var viewOpts = {
       descending : opts.descending
     };
+    opts.startkey = typeof opts.startkey  === 'undefined' ?
```

Comment:
Hm, that is another way to do it, and yeah, it is a bit shorter. :) I'm fine with either way, but @nicolasbrugneaux if you'd like to change it, that's cool too.